### PR TITLE
Add React/Vue snippets for Viz Series Types topics

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: concepts/05 Widgets/Chart/11 Series Types/10 Line Series.md [hunk 1]
```diff
@@ -68,6 +68,73 @@ To specify one or another series type, assign its name to the **series[]**.[type
...
+            <DxSeries ... />
+            ...
+            <DxCommonSeriesSettings
+                :line=" ... "
+                :spline=" ... "
+                :stepline=" ... "
+                :stackedline=" ... "
+                :stackedspline=" ... "
+                :fullstackedline=" ... "
+                :fullstackedspline=" ... "
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/10 Line Series.md]: It is better to declare the objects (`lineSettings`, `stackedSplineSettings`, etc.) in `data()` and bind the attributes to them. The same thing in React and in other similar cases in this PR.

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md [hunk 2]
```diff
@@ -46,6 +46,57 @@ Regulating the empty space on a bar's sides controls the relative bar width. Use
...
+    import Chart, {
+        Series,
+        CommonSeriesSettings
+    } from 'devextreme-react/chart';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ... >
+                    <Series barPadding={0.1} /> <!-- for an individual series -->
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]:
```suggestion
                    <Series barPadding={0.1} /> {/* for an individual series */}
```

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md [hunk 3]
```diff
@@ -46,6 +46,57 @@ Regulating the empty space on a bar's sides controls the relative bar width. Use
...
+                    <Series ... />
+                    ...
+                    <CommonSeriesSettings barPadding={0.1}/> <!-- for all series in the chart -->
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]: Please fix this in all React examples in this PR.

```suggestion
                    <CommonSeriesSettings barPadding={0.1}/> {/* for all series in the chart */}
```

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md [hunk 4]
```diff
@@ -84,6 +135,45 @@ The parent group's width can be specified using the [barGroupPadding](/api-refer
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :bar-group-padding="0.2"
+        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]:
```suggestion
            :bar-group-padding="0.2">
```

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md [hunk 5]
```diff
@@ -84,6 +135,45 @@ The parent group's width can be specified using the [barGroupPadding](/api-refer
...
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Chart from 'devextreme-react/chart';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ...
+                    barGroupPadding={0.2}
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]:
```suggestion
                    barGroupPadding={0.2}>
```

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md [hunk 6]
```diff
@@ -85,4 +136,43 @@ Bar groups' width can be changed using the [barGroupWidth](/api-reference/20%20D
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :bar-group-width="250"
+        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md]:
```suggestion
            :bar-group-width="250">
```

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md [hunk 7]
```diff
@@ -85,4 +136,43 @@ Bar groups' width can be changed using the [barGroupWidth](/api-reference/20%20D
...
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Chart from 'devextreme-react/chart';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart ...
+                    barGroupWidth={250}
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md]:
```suggestion
                    barGroupWidth={250}>
```

File: concepts/05 Widgets/Chart/11 Series Types/50 Bubble Series.md [hunk 8]
```diff
@@ -57,6 +57,71 @@ Note that **Bubble** series require an additional data field that provides size
...
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart
+            :min-bubble-size="14"
+            :max-bubble-size="0.3"
+            ...
+        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/50 Bubble Series.md]:
```suggestion
        <DxChart ...
            :min-bubble-size="14"
            :max-bubble-size="0.3">
```

File: concepts/05 Widgets/Chart/11 Series Types/50 Bubble Series.md [hunk 9]
```diff
@@ -57,6 +57,71 @@ Note that **Bubble** series require an additional data field that provides size
...
+    } from 'devextreme-react/chart';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Chart
+                    minBubbleSize={14}
+                    maxBubbleSize={0.3}
+                    ...
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/11 Series Types/50 Bubble Series.md]:
```suggestion
                <Chart ...
                    minBubbleSize={14}
                    maxBubbleSize={0.3}>
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Chart/11 Series Types/10 Line Series.md (hunk 1 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/10 Line Series.md]: Fixed

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md (hunk 2 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]: Fixed

**Review by RomanTsukanov — APPROVED** [review]

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md (hunk 3 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]: Fixed

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md (hunk 4 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]: Fixed

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md (hunk 5 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/01 Relative Bar Width.md]: Fixed

File: concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md (hunk 6 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/11 Series Types/20 Bar Series/05 Specify the Bar Width/02 Fixed Bar Width.md]: Fixed

**Pull request merged**